events/iisc_pulse_events.txt

```diff
@@ -101,23 +106,33 @@ country_event = {
 	}
 }
 
-# monthly pulse
+# monthly pulse but fires itself
 # trigger event for corps randomly in the month
-event = {
+# hq set
+country_event = {
 	id = iisc_pulse.20
 	is_triggered_only = yes
 	hide_window = yes
 	immediate = {
 		if = {
 			limit = {
-				NOT = {
-					has_global_flag = iisc_paused
+				OR = {
+					NOT = {
+						exists = event_target:iisc_hq_planet
+					}
+					has_global_flag = iisc_stopped
 				}
 			}
-			every_country = {
+			# TODO destroy the country and cleanup
```

Comment:
Do we need todo here? We have the effect to destroy the country

Reply:
Looks like I forgot it haha